examples/cpp/function-execution/main.cpp

```diff
@@ -0,0 +1,191 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * The Execute Function QuickStart Example.
+ *
+ * This example takes the following steps:
+ *
+ * 1. Create a Geode Cache.
+ * 2. Create the example Region Programmatically.
+ * 3. Populate some objects on the Region.
+ * 4. Create Execute Objects
+ * 5. Execute Functions
+ * 6. Close the Cache.
+ *
+ */
+
+// Include the Geode library.
+#include <iostream>
+#include <memory>
+
+#include <geode/CacheFactory.hpp>
+#include <geode/PoolManager.hpp>
+#include <geode/RegionFactory.hpp>
+#include <geode/RegionShortcut.hpp>
+#include <geode/FunctionService.hpp>
+#include <geode/CacheableString.hpp>
+
+// Use the "geode" namespace.
+using namespace apache::geode::client;
+
+const auto getFuncIName = std::string("MultiGetFunctionI");
+const auto putFuncIName = std::string("MultiPutFunctionI");
+const auto getFuncName = std::string("MultiGetFunction");
+const auto putFuncName = std::string("MultiPutFunction");
+
+// The Execute Function QuickStart example.
+int main(int argc, char** argv) {
```

Comment:
Yes it would, but sadly we can't, or at least shouldn't.  The purpose of doing it this way is to provide documentation in the code of what is required to update your app to work with the new version.  The original quickstart code has this as a long function, so we've left it that way to allow for a simple diff.  If we refactor into smaller methods, we'll lose the simplicity.